spring-cloud-stream/src/main/java/org/springframework/cloud/stream/binding/ChannelBindingService.java

```diff
@@ -57,13 +62,18 @@ public ChannelBindingService(ChannelBindingServiceProperties channelBindingServi
 		this.binderFactory = binderFactory;
 	}
 
-	public Binding<MessageChannel> bindConsumer(MessageChannel inputChannel, String inputChannelName) {
+	public Collection<Binding<MessageChannel>> bindConsumer(MessageChannel inputChannel, String inputChannelName) {
 		String channelBindingTarget = this.channelBindingServiceProperties.getBindingDestination(inputChannelName);
```

Comment:
Do we require to change this property/method name to destination**s**/getBindingDestination**s**? But, the `destination` doesn't differentiate whether it is for `@Output` or `@Input` type and I believe we won't support multiple destination names for `@Output` type. What do you think?